docker-compose-production.yml

```diff
@@ -9,6 +9,11 @@ services:
       dockerfile: docker/prod/django/Dockerfile
     ports:
       - "8000:8000"
+    logging:
+      driver: awslogs
+      options:
+        awslogs-region: ${AWS_DEFAULT_REGION}
+        awslogs-group: "django_production"
```

Comment:
I think we also need to add this option here: https://docs.docker.com/config/containers/logging/awslogs/#awslogs-create-group

Same for all below options

Reply:
I've created those log groups on AWS, since it's not dynamic. We don't need the option here.

Reply:
But what if someone forks EvalAI and tries to deploy in the production setting or we change our AWS account then we will have to create them manually. What is the issue in adding this option?

Reply:
Right, I didn't realize that. I'll add the option.

Reply:
Thanks!